{"name": "test_mp_article", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <scripts.test02_mp_article.TestMpArticle object at 0x048157A8>, expect = '操作成功'\n\n    def test_mp_article(self, expect=\"操作成功\"):\n        # 调用发布文章业务\n>       self.mp_article.page_mp_publish_article(title=\"文章小标题测试\", content=\"你们好啊，正在测试\")\n\nscripts\\test02_mp_article.py:27: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage\\page_mp_article.py:48: in page_mp_publish_article\n    self.page_click_add_content()\npage\\page_mp_article.py:12: in page_click_add_content\n    self.base_click(page.mp_add_article_text)\nbase\\base.py:45: in base_click\n    self.base_find_element(loc).click()\nbase\\base.py:34: in base_find_element\n    return (WebDriverWait(self.driver,\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e661bd8d7993e7c193f08cb76c173cbc\")>\nmethod = <function Base.base_find_element.<locals>.<lambda> at 0x043AD4F0>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\ne:\\users\\lxz\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium-3.141.0-py3.8.egg\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "attachments": [{"name": "log", "source": "c33cef43-c90f-4fd7-ae62-1beb8f188d0f-attachment.txt", "type": "text/plain"}], "start": 1616247515335, "stop": 1616247557038, "uuid": "1a8f4bea-81fe-4d19-9ef4-21347ed71e9d", "historyId": "7561356581f6e31adae19866f359262f", "testCaseId": "64196c9c705403acf80282af53eedc5f", "fullName": "scripts.test02_mp_article.TestMpArticle#test_mp_article", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test02_mp_article"}, {"name": "subSuite", "value": "TestMpArticle"}, {"name": "host", "value": "LAPTOP-PHNSP5RJ"}, {"name": "thread", "value": "26300-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test02_mp_article"}]}